globalcombined.lua

```diff
@@ -283,18 +340,42 @@ function et_RunFrame( leveltime )
 	if ( infocheck_client >= tonumber(et.trap_Cvar_Get("sv_maxclients")) ) then
 		infocheck_client = 0
 	end
+
 end
 
+
 function et_ClientUserinfoChanged( cno )
 --	printf("clientuserinfochanged %d\n",cno)
 	local reason = check_userinfo( cno )
+	local guid = string.upper(et.Info_ValueForKey(et.trap_GetUserinfo(cno), "cl_guid"))
 	if ( reason ) then
 		et.G_LogPrint(string.format("userinfocheck infochanged: client %d bad userinfo %s\n",cno,reason))
 		et.trap_SetUserinfo( cno, "name\\badinfo" )
 		et.trap_DropClient( cno, "bad userinfo", 0 )
 	end
+	for client = 0, (maxclients - 1) do

```

Comment:
This was created to try and avoid nick/guid stealing however did not always function correctly (according to micha, original author). It is likely this isn't really required for ETPro because of the lack of advantage gained by stealing these values.